### PR TITLE
Add accessiblity into framework more explicitly

### PR DESCRIPTION
**Pull request by alinaboghiu**
Co-authored by: @KarenFrankel9 

## What does this change?

Adds accessibility requirements into the performance framework more explicitly and at each level.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,7 +225,7 @@ Futher detail [here (staff only)](https://docs.google.com/document/d/1muN4pFqRhL
         <li>Aligns what they are working on to their team’s goals. </li>
         <li>Regularly gives useful feedback to more senior colleagues. </li>
         <li>Seeks to fully understand the end goal of the tasks they pick up, and is mindful of this throughout the delivery lifecycle.</li>
-        <li>Provides input into architectural design choices. </li>
+        <li>Provides input into architectural design choices </li>
```
Comment (alinaboghiu) [inline README.md:228]:
```suggestion
        <li>Provides input into architectural design choices. </li>
```

**Review by mxdvl — APPROVED** [review]
Love to see it!

**Review by bryophyta — APPROVED** [review]
👍👍

**Review by SarahJessica — APPROVED** [review]
Looks good to me.

**Pull request merged**